.evergreen/config.yml

```diff
@@ -77,61 +77,16 @@ functions:
 
   "create packages and repos":
     - command: "shell.exec"
+      shell: bash
       params:
-        script: |-
-          set -o xtrace
-          set -o errexit
-
-          # Find cmake.
-          if [ "$OS" == "Windows_NT" ]; then
-             CMAKE=/cygdrive/c/cmake/bin/cmake
-          else
-             # Amazon Linux 2 (arm64) has a very old system CMake we want to ignore
-             IGNORE_SYSTEM_CMAKE=1 . ./libmongocrypt/.evergreen/find-cmake.sh
-          fi
-          # Get current version of libmongocrypt.
-          cd libmongocrypt
-          mongocrypt_version="$($CMAKE -DRELEASE_BRANCH_REF=origin/r1.5 -P ./cmake/GetVersion.cmake 2>&1)"
-          cd ..
-
-          if [ "${has_packages|}" != "true" ] ; then
-            echo "'has_packages' is not 'true', skipping package build"
-            exit 0
-          fi
-
-          # check if virtualenv is set up
-          if [ -d "${workdir}/venv" ]; then
-            if [ "Windows_NT" = "$OS" ]; then
-              # Need to quote the path on Windows to preserve the separator.
-              . "${workdir}/venv/Scripts/activate" 2> /tmp/activate_error.log
-            else
-              . ${workdir}/venv/bin/activate 2> /tmp/activate_error.log
-            fi
-            if [ $? -ne 0 ]; then
-              echo "Failed to activate virtualenv: $(cat /tmp/activate_error.log)"
-            fi
-            python=python
-          else
-            python=${python|/opt/mongodbtoolchain/v3/bin/python3}
-          fi
-
-          if [ "Windows_NT" = "$OS" ]; then
-            export PYTHONPATH="$PYTHONPATH;$(cygpath -w ${workdir}/src)"
-          else
-            export PYTHONPATH="$PYTHONPATH:${workdir}/src"
-          fi
-
-          PPA_BUILD_ONLY=1 ${compile_env|} ./libmongocrypt/.evergreen/build_all.sh
-          pkg_version=$mongocrypt_version
-          tar -zcv -C install --transform="s|^libmongocrypt/|libmongocrypt-$pkg_version/|" --exclude=nocrypto --exclude=sharedbson -f libmongocrypt-$pkg_version.tar.gz libmongocrypt
-          cd libmongocrypt/etc/
-          # The files from libmongocrypt/debian/ are the official maintainer scripts,
-          # but libmongocrypt/etc/debian/ contains a few custom scripts that are
-          # meant to support the packager.py workflow.  This step "fills in" around
-          # those custom scripts.
-          cp -nr ../debian/* debian/
-          $python ./packager.py --prefix `pwd`/.. --distros ${packager_distro} --tarball `pwd`/../../libmongocrypt-$pkg_version.tar.gz --library-version $pkg_version --metadata-gitspec HEAD --arches ${packager_arch}
-          cd ../..
+        script: |
+          env "WORKDIR=${workdir}" \
+              "PYTHON=${python|}" \
+              "HAS_PACKAGES=${has_packages|false}" \
+              "PACKAGER_DISTRO=${packager_distro}" \
+              "PACKAGER_ARCH=${packager_arch}" \
+              ${compile_env|} \
+            bash libmongocrypt/.evergreen/create-packages-and-repos.sh
```

Comment:
Trivial: the indentation looks a little off here.

Reply:
The indentation is intentionally strange here because it's all a single `env <vars> <cmd>` command, and the whitespacing is meant to emphasize it.

Reply:
Well, it definitely jumped out at me, so mission accomplished! :rofl: